### DISCUSSION TITLE
No more updates coming out

### DISCUSSION THREAD

**Discussion by pl4udible** [3kh0/ext-remover#268] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
Echo isn't the dev of the script ,and the original dev made a post here https://github.com/FogNetwork/Ingot/issues/13 basically saying that he won't be able to fix the exploit for users on 106 or above.

Sorry Lads, credit to the guy that send it

**Comment by 3kh0** [comment]
no shit Sherlock, i said many times, i am not the dev

**Reply by piplup7575** [reply]
Nebelung is not the original dev 💀
Bypassi on top1!1!!!

**Reply by Aka-but-nice** [reply]
Does he have a discord server?

**Comment by md4r2** [comment]
[Pingot](https://github.com/Holding-THE-CREATOR/Pingot) is coming in around a month.

**Reply by Gumkid3** [reply]
yay?

**Reply by 3kh0** [reply]
thats was put out a month ago...

**Reply by 1-Mythical** [reply]
Pingot is such a random repository. What are the chances of it working anyways. My school blocked javascript on chrome.google.com/webstorex anyways, so unless Pingot fixes that, I am not interested. Good luck to devs to make it work.

**Reply by Aka-but-nice** [reply]
true but who knows

**Reply by 3kh0** [reply]
Pingot is a scam, it is not gonna work

**Reply by The-Man-Is-Me** [reply]
Do you know this for certain?

**Reply by 3kh0** [reply]
It said it needed a month almost 3 months ago, the so called dev behind it is a no body, stop playing yourself

**Reply by Aka-but-nice** [reply]
FAX

**Comment by Aka-but-nice** [comment]
There is a update #348 #347
